content/en/get-started/store-and-retrieve/troubleshooting/index.md

```diff
@@ -23,19 +23,20 @@ Reset your Lotus full-node computer if you are still facing issues after editing
 
 Run `echo $SHELL` to find out which shell you are using:
 
-```shell with-output
-echo $SHELL
-```
-```                                                                                                ~
-/bin/zsh
-```
+    ```shell
+    echo $SHELL
+    ```
+
+    ```plaintext
+    /bin/zsh
+    ```
 
 ## Which CPU does my Mac have?
 
 Open a terminal window and run:
 
-```shell
```

Comment:
This looks not right when I test it. 
It look like 
```
    ```shell
    sysctl -n machdep.cpu.brand_string
    ```
```